store/tikv/coprocessor.go

```diff
@@ -497,12 +507,28 @@ func (worker *copIteratorWorker) run(ctx context.Context) {
 		if respCh == nil {
 			respCh = task.respChan
 		}
-
 		worker.handleTask(ctx, task, respCh)
 		close(task.respChan)
-		if worker.respChan != nil {
+
+		worker.actionOnExceed.cond.L.Lock()
```

Comment:
Can we define a member func for actionOnExceed like 
``` go
destroyTokenIfNeeded(maxID, func())
```

Then we can invoke `destroyTokenIfNeeded(task.id, worker.sendRate.putToken)` here, it's more readable.